GM/gmpart.py

```diff
@@ -36,6 +36,20 @@ def _all_strings(self, strip=False, types=(NavigableString, CData)):
             yield descendant
 
 
+# decorator  
```

Comment:
In my opinion, this comment is not necessary. It doesn't actually describe the function. If it is - please put it inside the function. If you just want to leave it as is - remove 2 trailing spaces (PEP8 W291)

Reply:
I agree with @moxxiq .